src/tools/rbd/action/MirrorPool.cc

```diff
@@ -399,6 +503,10 @@ Shell::Action action_info(
   {"mirror", "pool", "info"}, {},
   "Show information about the pool mirroring configuration.", {},
   &get_info_arguments, &execute_info);
+Shell::Action action_status(
+  {"mirror", "pool", "status"}, {},
```

Comment:
It's probably worth having a general summary command (or option for this command), that doesn't list the details for each image, but just a high-level status like OK - images are all within some threshold, WARN if some images are out of sync by more than some threshold, and ERR if info is stale or any images are stuck (split-brain, or maybe just behind by a larger threshold)?